frontend/src/app/supportAdmin/DeviceType/DeviceForm.tsx

```diff
@@ -102,12 +104,15 @@ const DeviceForm = (props: Props) => {
               {props.deviceOptions ? (
                 <div className="grid-row grid-gap">
                   <div className="tablet:grid-col">
-                    <Select
-                      label="Select device"
+                    <label className="usa-legend" htmlFor="selectDevice">
+                      Select Device{" "}
+                      <span className="text-secondary-vivid">*</span>
```

Comment:
Instead of putting the * star here, you could use `<Required label={"Select device} />` from the [Required component](https://github.com/CDCgov/prime-simplereport/blob/main/frontend/src/app/commonComponents/Required.tsx)

Reply:
ooh thank you @BobanL for the suggestion, I did not know about that!

Reply:
done! pls take a look back :)